apps/stats-mainnet/src/styles/styles.css

```diff
@@ -0,0 +1,31 @@
+@tailwind base;
+@tailwind components;
+
+@font-face {
+  font-family: 'AlphaLyrae-Medium';
+  src: url('./AlphaLyrae-Medium.woff');
```

Comment:
follow up (non-blocking): we need to add alpha lyrae support to the UI toolkit

Reply:
Yes indeedy. Bart and I were talking about that this morning, we're going to find a way to have it accessible for the whole monorepo

Reply:
#110